app/controllers/admin/content/system-images/list.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+export default Controller.extend({
+  actions: {
+    openModal(selectedSubTopic) {
+      this.set('selectedSubTopic', selectedSubTopic);
+      this.set('isModalOpen', true);
```

Comment:
No ... Inside `modals/change-image-modal` ... alias `isOpen` to  `subTopic`. So, when ever `selectedSubTopic` is set, model will open